### PR TITLE
Added hyperlink to slide deck

### LINKED ISSUE #228: [Site] Add hyperlink to slide deck
**Current State of the service-mesh-performance site**
The project overview deck is iframe'd on the main project page - https://smp-spec.io
Visitors that want direct access to the presentation, though, aren't provided a link to the presentation.

**Desired State of the service-mesh-performance site**
Add a [hyperlink to the deck](https://docs.google.com/presentation/d/1fm7Eu1T3JSBPVmdFYyyevqQLSmAMLXqzWqC7QxTHGX8/edit#) underneath the iframe'd display of the deck.

### PR DESCRIPTION
**Pull request by soham4abc**
Signed-off-by: Soham Banerjee <sohambanerjee4abc@hotmail.com>

**Description**
Added hyperlink to slide underneath the preview
This PR fixes #228

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by warunicorn19 — APPROVED** [review]
LGTM🎉, thanks @soham4abc

**Comment by soham4abc** [comment]
@leecalcote please do have a look!

**Comment by leecalcote** [comment]
Thank you, @soham4abc. A small request: will you place this outside of the modal or will you ensure that the link changes color or activates in some way?

**Comment by leecalcote** [comment]
While you're in here, will you add links to the other slides, too?

- https://calcotestudios.com/talks/decks/slides-kubecon-eu-2020-discreetly-studying-the-effects-of-individual-traffic-control-functions.html
- https://calcotestudios.com/talks/decks/slides-kubecon-eu-2019-service-meshes-at-what-cost.html

**Comment by soham4abc** [comment]
> Thank you, @soham4abc. A small request: will you place this outside of the modal or will you ensure that the link changes color or activates in some way?

I could not get your point! 
"Activates in some other way" means?

**Comment by warunicorn19** [comment]
Hey @soham4abc, so basically you would have to add like a hover effect to the link to make it understandable that it's clickable, or make it a specific colour and give it an underline to make the user understand that this can be clicked.

**Comment by soham4abc** [comment]
> Hey @soham4abc, so basically you would have to add like a hover effect to the link to make it understandable that it's clickable, or make it a specific colour and give it an underline to make the user understand that this can be clicked.

done please have a look

**Comment by soham4abc** [comment]
> While you're in here, will you add links to the other slides, too?
> 
> * https://calcotestudios.com/talks/decks/slides-kubecon-eu-2020-discreetly-studying-the-effects-of-individual-traffic-control-functions.html
> * https://calcotestudios.com/talks/decks/slides-kubecon-eu-2019-service-meshes-at-what-cost.html

Should I do this in the same PR?

**Comment by warunicorn19** [comment]
Yes sir.

**Comment by soham4abc** [comment]
@leecalcote  @warunicorn19  All Done please do have a look !

**Comment by leecalcote** [comment]
Close, but no cigar. ;)

@NikhilSharma03, I wonder if you might offer some feedback.

**Comment by soham4abc** [comment]
> Close, but no cigar. ;)
> 
> @NikhilSharma03, I wonder if you might offer some feedback.

should I just keep the background completely rectangular then?

**Comment by NikhilSharma03** [comment]
@soham4abc Please avoid using inline styling, instead doing it in scss file would be better. I think we can give a bit more padding around the button and decrease the `border-radius` (4px looks good on my browser.) . Also, I think adding the hover effect would be an improvement.

**Comment by soham4abc** [comment]
> @soham4abc Please avoid using inline styling, instead doing it in scss file would be better. I think we can give a bit more padding around the button and decrease the `border-radius` (4px looks good on my browser.) . Also, I think adding the hover effect would be an improvement.

okay I am on it!

**Comment by soham4abc** [comment]
@NikhilSharma03 @leecalcote  Please do have a look!

**Comment by NikhilSharma03** [comment]
@soham4abc It looks better now. I wonder if we can add a bit of margin between slides and button in project section. @warunicorn19 What are your thoughts. Any other changes we can add?

**Comment by warunicorn19** [comment]
Yes, please add some margin between them and the yellow doesn't match our colour scheme, try using the yellow from the `SMP IN-ACTION` button.

**Comment by soham4abc** [comment]
Changes done as per request. Please do have a look! 
@leecalcote 
@warunicorn19 
@NikhilSharma03

**Review by debo19 — CHANGES_REQUESTED** [review]
Hey @soham4abc  thanks for the effort.Just some minor tweaks, then this should be good to go. Instead of just the text we can make the entire button have the hover effect similar to the `SMP in-action` button on the page. Also, you can remove the underline from the text.  And please move the button outside the presentation modal.

**Comment by soham4abc** [comment]
I am encountering a weird problem here. In order to remove underline `text-decoration:none` is not working!

**Comment by debo19** [comment]
@soham4abc removing the `<u></u>` tag should work

**Comment by soham4abc** [comment]
> @soham4abc removing the `<u></u>` tag should work

Thank you! Silly me :'-)

**Comment by soham4abc** [comment]
Changes done as per request. Please do have a look!
@leecalcote
@warunicorn19
@NikhilSharma03
@debo19

**Comment by warunicorn19** [comment]
Screenshot 2021-08-25 at 7 47 54 PM

I hate to keep doing this, but could please align these together, the same has happened with the slides below, if you could align them together, it would be great😃.

**Comment by soham4abc** [comment]
> Screenshot 2021-08-25 at 7 47 54 PM
> 
> I hate to keep doing this, but could please align these together, the same has happened with the slides below, if you could align them together, it would be great😃.

Sure no problem

**Comment by soham4abc** [comment]
@warunicorn19  It's done as per your request please do have a look!

**Comment by soham4abc** [comment]
@leecalcote  Please do have a look!

**Comment by leecalcote** [comment]
@soham4abc thanks for this reminder. I'm buried in GitHub notifications. // @Jashpatel1

**Comment by leecalcote** [comment]
@soham4abc if you will make the word "Slides" a hyperlink, I think this is all that we're looking for here.

**Comment by soham4abc** [comment]
> @soham4abc if you will make the word "Slides" a hyperlink, I think this is all that we're looking for here.

That means the `get presentations here` is not needed right?

**Comment by soham4abc** [comment]
@leecalcote  Please do have a look please!

**Comment by soham4abc** [comment]
@leecalcote  Do have a look Please!

**Comment by soham4abc** [comment]
@leecalcote  any more changes required?

File: docs/_includes/about.html [hunk 1]
```diff
@@ -53,8 +53,12 @@ <h3>Project Overview</h3>
               <br />
               <div class="google-slides">
                 <iframe class= "slides" src="https://docs.google.com/presentation/d/e/2PACX-1vSOFepq8Cu65owD2q1wkmhKYSQ2Pur3OqnGz1naAFusEuIMK0PWQtNqp0RIoZleY7B4SP5u1cqwiOwH/embed?start=true&loop=true&delayms=5000" frameborder="0" width="960" height="577" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
-              </div>            
+              </div>
+              <div style="padding-top: 5px;">
+              <a class="btn-slides" href="https://docs.google.com/presentation/d/1fm7Eu1T3JSBPVmdFYyyevqQLSmAMLXqzWqC7QxTHGX8/edit#" target="_blank" >Get the presentation here</a>
```
Comment (leecalcote) [inline docs/_includes/about.html:58]:
```suggestion
              <a class="btn-slides" href="https://docs.google.com/presentation/d/1fm7Eu1T3JSBPVmdFYyyevqQLSmAMLXqzWqC7QxTHGX8/edit#" target="_blank" >Slides</a> 
```

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
Thank you, @soham4abc 🥇

**Pull request merged**